### ISSUE TITLE
Staging Review finding: Inconsistent focus management when moving between pages.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#88739]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Benefits team 1
**Product:** Claim status tool
**Feature:** Claims process overview page

## Findings details
**VA.gov Experience Standard - issue:** [User encounters design components or patterns that are inconsistent or confusing.](https://design.va.gov/about/experience-standards/#user-encounters-consistent-clear-design-components)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

This is a repeat from the last staging review with this team, logged in #80606. When moving between the Status, Files, and Overview pages of each claim, sometimes keyboard focus moves to the H2 describing the page, and sometimes it moves to the breadcrumbs at the top of the page.

As discussed before, most users should still be able to accomplish tasks, but the "I never know where I'm going to end up" feeling could be frustrating to keyboard-only users and especially screen reader users. Consistent behavior here would be great.

## Recommended action

It looks like there's work currently being tracked in #80606, and you're working through some technical hurdles. No specific recommendation beyond encouraging that this be prioritized.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Keyboard
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by bacitracin** [comment]
https://github.com/department-of-veterans-affairs/va.gov-team/issues/80606 which this is a dupe of has been closed, this ticket can be closed as well.
